x-pack/solutions/search/plugins/enterprise_search/server/deprecations/pre_eight_index_deprecator.ts

```diff
@@ -24,9 +30,10 @@ export async function getPreEightEnterpriseSearchIndices(
     expand_wildcards: ['all'],
   });
 
-  if (!entSearchIndices) {
-    return [];
-  }
+  const entSearchDatastreams = await esClient.indices.getDataStream({
+    name: ENT_SEARCH_DATASTREAM_PATTERN.join(','),
+    expand_wildcards: ['all'],
```

Comment:
```suggestion
    expand_wildcards: ['all' ,'hidden'],
```

Reply:
Doesn't `all` include hidden? Or is that only for getting indices?